api-operator/pkg/registry/https_test.go

```diff
@@ -24,6 +22,9 @@ func TestGetPathWithoutReg(t *testing.T) {
 
 	for _, data := range datas {
 		result := getPathWithoutReg(data.in)
-		assert.Equal(t, result, data.out)
+		t.Log("Function Executing!!!")
+		if result != data.out {
+			t.Error("Paths do not match!!")
```

Comment:
Shall we add the actual value and wanted value in the error description.
Like "Registry image path without registry host is not matching, getPathWithoutReg(%v) = %v, want = %v"

sample: https://github.com/wso2/k8s-api-operator/blob/9b9e40a1e7e7385d2dc917f1c25b69babbf076bd/api-operator/pkg/apiproject/status/project_test.go#L58